packages/apps/transfer/src/services/key-pairs/key-pairs.ts

```diff
@@ -0,0 +1,12 @@
+import { genKeyPair } from '@kadena/cryptography-utils';
+
+import { downloadFileToBrowser } from '../utils/file-download';
+
+export function downloadKeyPairToBrowser(): void {
+  const keyPair = genKeyPair();
+
+  downloadFileToBrowser(
+    'key-pair.txt',
+    `public: ${keyPair.publicKey}\r\nprivate: ${keyPair.secretKey}`,
+  );
+}
```

Comment:
Idea: could we change the textfile to be `kadena-<pubkey>.txt`? That way we don't inadvertently overwrite an existing key